File: Baekjoon/브루트포스/11399.py
# https://www.acmicpc.net/problem/11399

import sys
input = sys.stdin.readline

n = int(input())
arr = list(map(int, input().split(" ")))
result = 0
arr.sort()

for i in range(n):
    for j in range(i + 1):
        result += arr[j]
print(result)

File: Baekjoon/Greedy/11399.cpp
// https://www.acmicpc.net/problem/11399

#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int N;
    int arr[1000];
    int total = 0;

    cin >> N;
    for (int i = 0; i < N; i++) {
        cin >> arr[i];
    }

    sort(arr, arr + N);
    for (int i = 0; i < N; i++) {
        total += arr[i] * (N - i);
    }

    cout << total << endl;

    return 0;
}